docs/fuel-101/sequencer.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Sequencer
+category: Fuel 101
+---
+
+# What is a Sequencer?
+
+> A sequencer is a component utilized in Layer 2 scaling solutions, such as Optimistic Rollups and certain types of Zero-Knowledge Rollups to order transactions.
+
+Since rollups strategically leverage the consensus and data availability of another layer 1 chain, they depend on a sequencer for transaction ordering.
+
+By efficiently ordering transactions, sequencers allow Layer 2 solutions to significantly reduce transaction processing times and costs. This efficiency gain is essential for scaling blockchain networks and improving their usability for a wider range of applications, making the trade-off an integral part of optimizing blockchain performance.
+
+## Shared Sequencer
+
+> A shared sequencer model involves multiple entities or nodes taking turns or sharing the responsibility of sequencing transactions.
+
+Fuel leverages a shared sequencer an Application Blockchain Interface (ABCI), integrated with the Cosmos network. Validators are responsible for
+
+1. Establishing the order of Fuel's blocks
+2. Verifying these optimistically ordered blocks through fraud proofing
+
+Any misbehaving actors producing bad blocks within the system are penalized (slashed) on Ethereum. The sequencer network uses the `secp256k1` cryptographic algorithm, which is compatible with Ethereum's infrastructure.
+
+### 1. Transaction Batching
+
+The shared sequencer network to collect a large number of transactions over a certain period. Instead of each transaction being submitted individually to the base chain, the sequencer batches these transactions into a single package. This approach significantly reduces the number of transactions that the base chain has to process, as a single batched transaction on the base chain can represent hundreds or thousands of individual transactions.
```

Comment:
```suggestion
The shared sequencer network collects and gathers a large number of transactions over a certain period of time. Instead of each transaction being submitted individually to the base chain, the sequencer batches these transactions into a single package. This approach significantly reduces the number of transactions that the base chain has to process, as a single batched transaction on the base chain can represent hundreds or thousands of individual transactions.
```